lib/virtfs/block_io.rb

```diff
@@ -11,6 +11,8 @@ module VirtFS
   class BlockIO
     MIN_BLOCKS_TO_CACHE = 64
 
+    attr_reader :seek_pos
```

Comment:
@movitto This interface implements the standard block IO interface (read, write and seek). There should be no  need for this accessor, given seek returns the current position and seeks can be relative.